core/tx_journal.go

```diff
@@ -57,12 +57,12 @@ func newTxJournal(path string) *txJournal {
 // load parses a transaction journal dump from disk, loading its contents into
 // the specified pool.
 func (journal *txJournal) load(add func([]*types.Transaction) []error) error {
-	// Skip the parsing if the journal file doesn't exist at all
-	if !common.FileExist(journal.path) {
-		return nil
-	}
 	// Open the journal for loading any past transactions
 	input, err := os.Open(journal.path)
+	if os.IsNotExist(err) {
```

Comment:
Thanks, this looks good, but the method doc says it's deprecated and has a suggestion for how to do it in new code:

```
This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist).
```

https://pkg.go.dev/os#IsNotExist

Reply:
According to the cause of deprecated, I think it could be use the `os.ErrNotExist` without import `fs`.
(https://github.com/golang/go/issues/38198)

```
// os package
ErrNotExist = fs.ErrNotExist // https://pkg.go.dev/os#pkg-variables
```

Using `fs.ErrNotExist` with `fs` import or using `os.ErrNotExist`, which one better?

Reply:
I have applied that !